all-the-stars-js/js/model.js

```diff
@@ -120,12 +126,30 @@ model.saveToCloud = function(callback) {
   });
 };
 
-
-
 model.getStarsFor = function(world, level) {
   return model.inv.getStarsFor(world, level);
 };
 
 model.setStarsFor = function(world, level, newNum){
   model.inv.setStarsFor(world, level, newNum);
+  // Activate achievements when the user reaches a predetermined number of 
+  // stars.  Mapping achievement IDs to more meaningful constants for 
+  // readability is left as an exercise for the reader.
+  switch (model.inv.countMyStars()) {
+  case 8:
+    pgs.achievements.unlock('CgkImq-v4t4UEAIQAQ');
```

Comment:
can these constants be moved to another file.  That way they can be replaced when running the sample.